config/clients/python/config.overrides.json

```diff
@@ -2,7 +2,7 @@
   "sdkId": "python",
   "gitRepoId": "python-sdk",
   "packageName": "openfga_sdk",
-  "packageVersion": "0.4.3",
+  "packageVersion": "0.4.4",
```

Comment:
Could we potentially bump to `v0.5.0` similar to the other SDKs, that way we're indicating the breaking change in the `list_users` API.